geopandas/tests/test_op_output_types.py

```diff
@@ -144,6 +145,32 @@ def test_loc(df):
     assert_object(df.loc[:, "value1"], pd.Series)
 
 
+@pytest.mark.parametrize(
+    "geom_name",
+    [
+        "geometry",
+        pytest.param(
+            "geom",
+            marks=pytest.mark.xfail(
+                reason="pre-regression behaviour only works for geometry col geometry"
+            ),
+        ),
```

Comment:
With black's formatting it's a bit verbose, but moved the xfail here. The advantage of using a marker instead of calling `pytest.xfail` inside the test is that this way the test will actually run and check it fails, and so will also notify us if the test starts passing

Reply:
I did this at first too, but was not thrilled by the formatting. But maybe this will xpass one day and it will be useful

Reply:
Yes, that's the reason I prefer this, so we remember to update this test if at some point it gets fixed ;)